test/unit/acc/src/AccTagTest.cpp

```diff
@@ -0,0 +1,218 @@
+    alpaka::AccCpuFibersTag,
+    alpaka::AccCpuTbbBlocksTag,
+    alpaka::AccCpuOmp2BlocksTag,
+    alpaka::AccCpuOmp2ThreadsTag,
+    alpaka::AccOmp5Tag,
+    alpaka::AccOaccTag,
+    alpaka::AccGpuCudaRtTag,
+    alpaka::AccGpuHipRtTag,
+    alpaka::AccCpuSyclIntelTag,
+    alpaka::AccFpgaSyclIntelTag,
+    alpaka::AccFpgaSyclXilinxTag,
+    alpaka::AccGpuSyclIntelTag>;
+
+using AccTagTuple = std::tuple<
+    std::tuple<alpaka::test::detail::AccCpuSerialIfAvailableElseInt<Dim, Idx>, alpaka::AccCpuSerialTag>,
+    std::tuple<alpaka::test::detail::AccCpuThreadsIfAvailableElseInt<Dim, Idx>, alpaka::AccCpuThreadsTag>,
+    std::tuple<alpaka::test::detail::AccCpuFibersIfAvailableElseInt<Dim, Idx>, alpaka::AccCpuFibersTag>,
+    std::tuple<alpaka::test::detail::AccCpuTbbIfAvailableElseInt<Dim, Idx>, alpaka::AccCpuTbbBlocksTag>,
+    std::tuple<alpaka::test::detail::AccCpuOmp2BlocksIfAvailableElseInt<Dim, Idx>, alpaka::AccCpuOmp2BlocksTag>,
+    std::tuple<alpaka::test::detail::AccCpuOmp2ThreadsIfAvailableElseInt<Dim, Idx>, alpaka::AccCpuOmp2ThreadsTag>,
+    std::tuple<alpaka::test::detail::AccOmp5IfAvailableElseInt<Dim, Idx>, alpaka::AccOmp5Tag>,
+    std::tuple<alpaka::test::detail::AccOaccIfAvailableElseInt<Dim, Idx>, alpaka::AccOaccTag>,
+    std::tuple<alpaka::test::detail::AccGpuCudaRtIfAvailableElseInt<Dim, Idx>, alpaka::AccGpuCudaRtTag>,
+    std::tuple<alpaka::test::detail::AccGpuHipRtIfAvailableElseInt<Dim, Idx>, alpaka::AccGpuHipRtTag>,
+    std::tuple<alpaka::test::detail::AccCpuSyclIntelIfAvailableElseInt<Dim, Idx>, alpaka::AccCpuSyclIntelTag>,
+    std::tuple<alpaka::test::detail::AccFpgaSyclIntelIfAvailableElseInt<Dim, Idx>, alpaka::AccFpgaSyclIntelTag>,
+    std::tuple<alpaka::test::detail::AccFpgaSyclXilinxIfAvailableElseInt<Dim, Idx>, alpaka::AccFpgaSyclXilinxTag>,
+    std::tuple<alpaka::test::detail::AccGpuSyclIntelIfAvailableElseInt<Dim, Idx>, alpaka::AccGpuSyclIntelTag>>;
+
+using AccTagTestMatrix = alpaka::meta::CartesianProduct<std::tuple, AccTagTuple, TagList>;
+
+TEMPLATE_LIST_TEST_CASE("relationAccTag", "[acc][tag]", AccTagTestMatrix)
+{
+    // type of TestType is std::tuple<std::tuple<TAcc, TTag>, TTag>
+    // TAcc is ether a alpaka accelerator or int, if not enabled
+    // The Tag in in the inner tuple is the expected Tag of the TAcc
+    // The Tag in the outer tuple is the Tag to test
+    using TestAccTestTuple = std::tuple_element_t<0, TestType>;
+    using TestAcc = std::tuple_element_t<0, TestAccTestTuple>;
+    using ExpectedTag = std::tuple_element_t<1, TestAccTestTuple>;
+    using TestTag = std::tuple_element_t<1, TestType>;
+
+    // if the Acc is not enabled, the type is int
+    if constexpr(!std::is_same_v<TestAcc, int>)
+    {
+        std::cout << "Tested Acc: " << alpaka::getAccName<TestAcc>() << "\n"
+                  << "Expected Tag: " << ExpectedTag::get_name() << "\n"
+                  << "Tested Tag: " << TestTag::get_name() << "\n"
+                  << std::endl;
```

Comment:
Please remove unnecessary output statements. They make the CI logs longer and errors harder to find.

Reply:
That's a mistake. It should be `INFO()` and only print something, if the test is going wrong. But on the other site, that's not possible, because the checks are static and throws and compile time error :thinking:

Reply:
I removed it